eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -242,9 +231,25 @@ function Upload-Blobs
         & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/latest$($SASKey)" --recursive=true
     }
 }
+
+function RetrieveReleaseTag([Object[]]$pkgs) {
```

Comment:
You can probably add this function to the shared metadata script. I would probably call it `Get-PackageReleaseTag` or something like that.

Reply:
It doesn't make too many senses to add this into shared scripts.
Verify-Package returns us a list of packages. Release tag associate with each package. So, taking a list has to return a list of release tag. If take a package, then we need to check if pkgList is empty before call `Get-PackageReleaseTag`. Not save any codes at all.

Reply:
I'm find with this being here until we make larger fixes to the github.io workflows. 

However I was suggesting we not have to call VerifyPackages directly here but instead call `Get-PackageReleaseTag` and it would do all the work.